### ISSUE TITLE
LOFTEE flags on variant page are illegible

### ISSUE THREAD

**Issue by nawatts** [macarthur-lab/gnomadjs#457]
LOFTEE flags are written in yellow text, which is very hard to read. The red text used for LC annotations doesn't meet WCAG AA contrast requirements either.

<img width="327" alt="Screen Shot 2019-08-15 at 1 36 43 PM" src="https://user-images.githubusercontent.com/1156625/63114003-be7b5f00-bf61-11e9-8063-b000d120b0b7.png">

**Comment by nawatts** [comment]
Polyphen and SIFT predictions have the same problem.
